data/json/mapgen/house/house33.json

```diff
@@ -0,0 +1,631 @@
+        "-": "t_concrete",
+        "$": "t_thconc_floor",
+        "q": "t_thconc_floor",
+        "U": "t_thconc_floor",
+        "(": "t_wall_r",
+        "0": "t_region_groundcover_urban",
+        "~": "t_floor_noroof",
+        "G": "t_floor_noroof",
+        "K": "t_floor_noroof",
+        "@": "t_carpet_yellow",
+        "D": "t_carpet_yellow",
+        "&": "t_carpet_yellow",
+        "d": "t_carpet_yellow",
+        "k": "t_carpet_yellow",
+        "I": "t_carpet_yellow",
+        "R": "t_carpet_yellow",
+        "]": "t_carpet_yellow",
+        "T": "t_carpet_yellow"
+      },
+      "furniture": {
+        "G": "f_deckchair",
+        "%": "f_makeshift_bed",
+        "&": "f_table",
+        "k": "f_chair",
+        "0": [
+          "f_black_eyed_susan",
+          "f_mutpoppy",
+          "f_lily",
+          "f_flower_tulip",
+          "f_datura",
+          "f_flower_spurge",
+          "f_dahlia",
+          "f_chamomile",
+          "f_sunflower",
+          "f_chicory",
+          "f_dandelion"
+        ]
+      },
+      "items": {
+        "~": { "item": "trash", "chance": 3 },
+        "&": { "item": "nightstand", "chance": 30 },
+        ")": { "item": "clutter_bathroom", "chance": 5 },
+        "[": { "item": "bedroom", "chance": 3 }
+      },
+      "place_loot": [ { "item": "television", "x": 13, "y": 12, "chance": 100 }, { "item": "lawnmower", "x": 2, "y": 3, "chance": 15 } ],
+      "place_vehicles": [ { "vehicle": "beetle", "x": 16, "y": 19, "chance": 20, "fuel": 30, "status": 50, "rotation": 270 } ],
+      "rotation": [ 2 ]
+    }
+  },
+  {
+    "//": "Ground Floor - Variant 2: more items ",
+    "type": "mapgen",
+    "om_terrain": "house_33",
+    "weight": 50,
+    "method": "json",
+    "object": {
+      "fill_ter": "t_floor",
+      "rows": [
+        "}}}}}}}}}}}}}}}}}}}}}}}}",
+        "}......................}",
+        "}.....u................}",
+        "}...=...|||~~~~~~~~~~..}",
+        "}......^|>*~~~GKG~~~~..}",
+        "}..||o|||||o||||oo*||..}",
+        "}..|@@&d[[(9St(ffh y|..}",
+        "}..|@@[[[k()))(ffh y|..}",
+        "}..o[[[[II()88(h    o..}",
+        "}..|[[[[(((+(((     o..}",
+        "}..|RRD[+       A  Y|..}",
+        "}..|(((((H l x3JJJ_F|..}",
+        "}..|dD[R(H l x2____6|..}",
+        "}..o[[[R(H l x15O7_6|..}",
+        "}..|%[[[+    y((((+(|..}",
+        "}..|%[[T(ii  L(qqq$U|..}",
+        "}..||o|||||o*o|$$$$$|..}",
+        "}......0000---|$$$$$|..}",
+        "}..=...0000---|$$$$$|..}",
+        "}.............|$$$$$|..}",
+        "}.............|$$$$$|..}",
+        "}........=....|?????|..}",
+        "}..=...........,,,,,/..}",
+        "}}}}}}}}}}}}p..,,,,,...}"
+      ],
+      "palettes": [ "standard_domestic_palette", "standard_domestic_lino_bathroom", "standard_domestic_lino_kitchen" ],
+      "terrain": {
+        "?": "t_door_metal_locked",
+        ",": "t_pavement",
+        "/": "t_gates_mech_control",
+        "=": [
```

Comment:
[ [ "t_region_tree_fruit", 2 ], [ "t_region_tree_nut", 2 ], "t_region_tree_shade" ] (any or all of these regional options for trees).